src/speech_to_text.cpp

```diff
@@ -87,16 +87,18 @@ bool vad_simple(std::vector<float> &pcmf32, int sample_rate, int last_ms, float
 	}
 
 	energy_all /= n_samples;
-	energy_last /= n_samples_last;
+	if (n_samples_last == 0) {
+		energy_last /= 1;
+	} else {
+		energy_last /= n_samples_last;
```

Comment:
can energy last ever be 0?

Reply:
Yes, if you pass in a completely blank audio segment, it may be 0.
Even if you haven't added any audio data to effect_capture, 
you always can get an audio array by effect_capture.get_buffer(cur_frames_available). But all these audio data are (0,0)